### ISSUE TITLE
Loss keeps increasing until it becomes 'nan'

### ISSUE THREAD

**Issue by xiaoyu0625** [MIVRC/MLEFGN-PyTorch#8]
When I was tring to train the model with gray-scaled images, I put the gray-scaled images and noisy images into folders of "DIV2K_train_HR" and "DIV2K_train_LR_bicubic/x1". The code seems running, however, the loss just keeps increasing during the training, and becomes "nan" finally after about 50 epochs of training. Maybe I have made something wrong about the training data? Can you provide some suggestions about this? Thanks.
